inc/class-registration.php

```diff
@@ -978,6 +980,58 @@ public static function sticky_style() {
 		echo '<style id="o-sticky-inline-css">.o-sticky.o-sticky-float { height: 0px; } </style>';
 	}
 
+	/**
+	 * Load the Hide on condition styles.
+	 *
+	 * @param string $block_content The block content.
+	 * @param array  $block The block data.
+	 * @return string
+	 */
+	public function load_condition_hide_on_styles( $block_content, $block ) {
+		if ( self::$scripts_loaded['condition_hide_on'] ) {
+			return $block_content;
+		}
+
+		if ( empty( $block['attrs']['otterConditions'] ) || ! is_array( $block['attrs']['otterConditions'] ) ) {
+			return $block_content;
+		}
+
+		$has_condition = false;
+
+		foreach ( $block['attrs']['otterConditions'] as $group ) {
+			foreach ( $group as $condition ) {
+				if ( 'screenSize' === $condition['type'] && is_array( $condition['screen_sizes'] ) ) {
```

Comment:
Similarly, we can also check here if screen_sizes exist before checking if it's an array.

Reply:
The `is_array` is already doing that job. If it does not exist it will return false.